docs-guides/source/convert-pytorch-workflow.md

```diff
@@ -33,64 +46,80 @@ out = traced_model(example_input)
 
 ## Convert to Core ML
 
-Convert the traced or scripted model to Core ML using the Unified Conversion API [`convert()`](https://apple.github.io/coremltools/source/coremltools.converters.convert.html#module-coremltools.converters._converters_entry) method. In the `inputs` parameter, you can use either [`TensorType`](https://apple.github.io/coremltools/source/coremltools.converters.mil.input_types.html#tensortype) or [`ImageType`](https://apple.github.io/coremltools/source/coremltools.converters.mil.input_types.html#coremltools.converters.mil.input_types.ImageType) as the input type.
+Convert the traced / exported model to Core ML using the Unified Conversion API [`convert()`](https://apple.github.io/coremltools/source/coremltools.converters.convert.html#module-coremltools.converters._converters_entry) method. In the `inputs` parameter, you can use either [`TensorType`](https://apple.github.io/coremltools/source/coremltools.converters.mil.input_types.html#tensortype) or [`ImageType`](https://apple.github.io/coremltools/source/coremltools.converters.mil.input_types.html#coremltools.converters.mil.input_types.ImageType) as the input type for the traced model.
 
-The following example uses `TensorType` and converts the PyTorch traced model to a [Core ML program](convert-to-ml-program) model. For more about image input conversions, see [Image Inputs](image-inputs).
+The following example uses `TensorType` and converts the PyTorch traced / exported model to a [Core ML program](convert-to-ml-program) model. For more about image input conversions, see [Image Inputs](image-inputs).
 
 ```python
 import coremltools as ct
 
-# Using image_input in the inputs parameter:
 # Convert to Core ML program using the Unified Conversion API.
-model = ct.convert(
+model_from_trace = ct.convert(
     traced_model,
+    inputs=[ct.TensorType(shape=example_input.shape)],
     convert_to="mlprogram",
-    inputs=[ct.TensorType(shape=example_input.shape)]
- )
+)
+model_from_export = ct.convert(
+    exported_program,
+    convert_to="mlprogram",
+)
 ```
 
 With the converted ML model in memory, you can save it as a [Core ML model package](convert-to-ml-program.md#save-ml-programs-as-model-packages):
 
 ```python
 # Save the converted model.
-model.save("newmodel.mlpackage")
+model_from_trace.save("newmodel_from_trace.mlpackage")
+model_from_export.save("newmodel_from_export.mlpackage")
 ```
 
 As an alternative, you can convert the model to a neural network by eliminating the `convert_to` parameter:
```

Comment:
I think we can get rid of this section now. We dont want to recommend neuralnetwork anymore (unless the user selects a low deployment target and automatically get an mlpackage with the nn format)

Reply:
Removed the neural network section as suggested